platform-coop-toolkit/coop-map-directory-index
# feat: add number of members to organizations (resolves #75)

## Description

Adds field for "Number of members" to Organization model.

## Steps to test

1. Check out this branch.
2. Run `./manage.py migrate mdi`.
3. Edit an organization via the admin user interface.

**Expected behavior:** A numeric field for Number of members is present.

## Additional information

Not applicable.

## Related issues

- Resolves #75.
- Depends on #80.
